components/common/FooterTwo.tsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+import { useScrollPosition } from "../../hooks/useScrollPosition";
+
+const FooterTwo = ({id}) => {
+
+  const [progress, setProgress] = useState(0); 
+  const { maxScroll, position, maxSize } = useScrollPosition(id);
+
+
+  const getScaledPosition = (position, maxScroll, maxSize) => {
+    return position * (maxSize / maxScroll);
+  }
+
+  useEffect(() => {
+    // console.log("maxScroll", maxScroll);
+    console.log("position", position);
+    let newPosition = getScaledPosition(position, maxScroll, maxSize);
+    if (newPosition <= 25) {
```

Comment:
Ah hell naw

Reply:
I know .... this hurt me too. Theres got to be a better way to handle this. Was doing so this way for testing purposes. This didn't even reliably remove the color as the event listener couldn't keep up with my SUPER FAST scrolling 🍆